### ISSUE TITLE
Tests for vector inequality are testing sequences

### ISSUE THREAD

**Issue by mratsim** [unicredit/linear-algebra#15]
In line 93 to 145 in equality, it's a seq that is tested not the vectors. Is that intended?
https://github.com/unicredit/linear-algebra/blob/master/tests/equality.nim#L93

**Comment by mratsim** [comment]
I see the same pattern in ops testing: https://github.com/unicredit/linear-algebra/blob/master/tests/ops.nim#L529

**Comment by andreaferretti** [comment]
Yup, dynamic vectors are in fact just seqs: https://github.com/unicredit/linear-algebra/blob/master/linalg/private/types.nim#L24-L25

I saw no good reason not to do this, and I thought that since seqs are so ubiquitous, this might save a few copies here and there
